hugo/layouts/partials/correction.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+
+
+<div class="columns container my-2">
+    <div class="column is-half is-offset-one-quarter">
+    <h1 class="title">Report errors in your paper's metadata</h1>
+
+  <form id="correction-form" method="post">
+    <input type="hidden" name="volume_id" value="{{ .volume_id }}">
+    <input type="hidden" name="anthology_id" value="{{ .anthology_id }}">
+    <input type="hidden" name="authors" value="{{ .paper.author}}">
+    <label for="paper_title">Paper title: </label>
+    <input type="text" id="paper_title" name="paper_title" value="{{ .paper.title }}"><br><br>
+    <label for="paper_url">Paper URL: </label>
+    <input type="text" id="paper_url" name="paper_url" value="{{ .paper.url }}"><br><br>
+    <label for="paper_author">Paper authors: </label>
+    {{ range $index, $element := .paper.author }}
+        {{ if $index }}
+          <label for="paper_author_first">First Name: </label>
+          <input type="text" id="paper_author_first_{{$index}}" name="paper_author_first_{{$index}}" value="{{ $element.first }}"><br><br>
+
+          <label for="paper_author_last">Last Name: </label>
+          <input type="text" id="paper_author_last_{{$index}}" name="paper_author_last_{{$index}}" value="{{ $element.last }}"><br><br>
+
+          <!-- Is full name always first + last name?  -->
+          <!-- <label for="paper_author_full">Full Name: </label>
+          <input type="text" id="paper_author_full_{{$index}}" name="paper_author_full_{{$index}}" value="{{ $element.full }}"><br><br> -->
+        {{ end }}
+      {{ end }}
```

Comment:
Two potential complications:
1. What do we do with names that require disambiguation through author IDs? We probably don't want to expose the complexity of author IDs to end users...
2. I understand that this is a prototype/proof-of-concept, but just a note that we’d ultimately need functionality to remove, add, and ideally reorder author fields as well.

Reply:
The ID could be a hidden field embedded into the web page. As for the aesthetics and design, I have a new mockup I'll post below.